modules/acs-requirements.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * installing/index.adoc
+:_module-type: CONCEPT
+[id="acs-general-requirements_{context}"]
+= General requirements
+
+To install {product-title}, you need:
+
+* {ocp} version {ocp-supported-version} or later.
++
+[WARNING]
+====
+You must not install {product-title} on:
+
+* Amazon Elastic File System (Amazon EFS). Use the Amazon Elastic Block Store (Amazon EBS) with the default *gp2* volume type instead.
+* Older CPUs that do not have the Streaming SIMD Extensions (SSE) 4.2 instruction set.
+For example, Intel processors older than _Sandy Bridge_ and AMD processors older than _Bulldozer_.
+(These processors were released in 2011.)
+====
+
+* Cluster nodes with a supported operating system.
+See the link:https://access.redhat.com/node/5822721[{product-title} Support Policy] for additional information.
+** *Operating system*: Amazon Linux, CentOS, Container-Optimized OS from Google, CoreOS, Debian, Red Hat Enterprise Linux, or Ubuntu.
+** *Processor and memory*: 2 CPU cores and at least 3GiB of RAM.
++
+[NOTE]
+====
+For deploying Central, use a machine type with 4 or more cores and apply scheduling policies to launch Central on such nodes.
+====
+
+* Persistent storage by using persistent volume claim (PVC).
+** Use Solid-State Drives (SSDs) for best performance.
+However, you can use another storage type if you do not have SSDs available.
+* Helm command-line interface (CLI) v3.2 or newer.
+Use the `helm version` command to verify the version of Helm you have installed.
+* The OpenShift CLI (`oc`).
```

Comment:
```suggestion
* The {ocp} CLI (`oc`).
```

Reply:
We actually do just call it the "OpenShift CLI" (and not the "OpenShift Container Platform CLI").